daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -28,6 +28,17 @@ There will be at least 6 weeks between major.minor version releases giving users
 
 Patch support is for supported versions (current and previous).
 
+## Build variations
+
+The Dapr's sidecar image is published to GitHub Container Registry and Docker Registry. The default image contains all components. Since version 1.11, Dapr also offers a variation of the sidecar image, containing only stable components.
+
+* Default sidecar images: `daprio/daprd:<version>` or `ghcr.io/dapr/daprd:<version>`
```

Comment:
```suggestion
* Default sidecar images: `daprio/daprd:<version>-<platform>` or `ghcr.io/dapr/daprd:<version>-<platform>`. For example, `daprio/daprd:1.11.0-linux-amd64`
```

Reply:
I may have this wrong above. My question is where does the tag `daprio/daprd:latest` come in here, since I presume this is the default image? Should we say something here?

Reply:
Platform is not required, since the -stable tag is multi-arch and K8s downloads the right arch automatically.